life/modernized/18-chapter-18.adoc

```diff
@@ -201,7 +206,8 @@ or flinging away all outward substance; but that your heart may reign above all
 and make God its treasure, and never stick in anything of this lower world,
 or rest short of Christ, the eternal rest of all the seed of faith.
 
-Here begins the narrative.
+[.alt.centered]
+==== Here begins the narrative.
```

Comment:
I wasn't sure how to label these, but this seemed best. I tried [.old-style] but it didn't feel right, but maybe that's just me.

Reply:
So I just thought of offsetting and italicizing these at the beginning and the end and I went back in and tried that and that looks even better I think. But whatever way you think is best!